requirements/AUR-CBI-INT-0010/component-management-behaviors/Makefile

```diff
@@ -0,0 +1,136 @@
+KAZOO?=kazoo
```

Comment:
This is applicable not to the file, but to the requirement. The validation method is M and I. M is satisfied. I do not see the expected test file with Inspection results.

Reply:
Done, added in .md file